### ISSUE TITLE
"Could not locate the required V8 native libraries" on macOS (.NET Core)

### ISSUE THREAD

**Issue by jeffijoe** [rjamesnw/v8dotnet#19]
Hi James! Thank you for this library!

I created a `dotnet new console` project using the .NET Core 2 as the target, and used `dotnet add package` to install V8.NET. This is on macOS.

However, I'm getting the `Could not locate the required V8 native libraries` error, even though I've checked the paths for the DLL, and it is indeed there.

Any ideas?

**Comment by rjamesnw** [comment]
While the Net Standard wrapper code will compile anywhere,  I did not compile any libraries for Mac because I don’t own one (nor would I know how to at the moment).  The V8 libraries that are included are only compiled for Windows PCs.  Someone would have to compile V8 on the Mac. If anyone is interested in taking up that task all they have to do is download the V8 source and compile it, or find the Mac compiled libraries online somewhere, download them, then give the the same names.  I did put in Support to load Linux and Mac libraries if found.
